src/rpc/rpc.cc

```diff
@@ -358,128 +329,128 @@ DGL_REGISTER_GLOBAL("distributed.server_state._CAPI_DGLRPCGetServerState")
 //////////////////////////// KVStore ////////////////////////////
 
 DGL_REGISTER_GLOBAL("distributed.rpc._CAPI_DGLRPCGetGlobalIDFromLocalPartition")
-.set_body([] (DGLArgs args, DGLRetValue* rv) {
-  NDArray ID = args[0];
-  NDArray part_id = args[1];
-  int local_machine_id = args[2];
-  int64_t* ID_data = static_cast<int64_t*>(ID->data);
-  int64_t* part_id_data = static_cast<int64_t*>(part_id->data);
-  int64_t ID_size = ID.GetSize() / sizeof(int64_t);
-  std::vector<int64_t> global_id;
-  for (int64_t i = 0; i < ID_size; ++i) {
-    if (part_id_data[i] == local_machine_id) {
-      global_id.push_back(ID_data[i]);
+  .set_body([](DGLArgs args, DGLRetValue* rv) {
+    NDArray ID = args[0];
+    NDArray part_id = args[1];
+    int local_machine_id = args[2];
+    int64_t* ID_data = static_cast<int64_t*>(ID->data);
+    int64_t* part_id_data = static_cast<int64_t*>(part_id->data);
+    int64_t ID_size = ID.GetSize() / sizeof(int64_t);
+    std::vector<int64_t> global_id;
+    for (int64_t i = 0; i < ID_size; ++i) {
+      if (part_id_data[i] == local_machine_id) {
+        global_id.push_back(ID_data[i]);
+      }
     }
-  }
-  NDArray res_tensor = dgl::aten::VecToIdArray<int64_t>(global_id);
-  *rv = res_tensor;
-});
+    NDArray res_tensor = dgl::aten::VecToIdArray<int64_t>(global_id);
+    *rv = res_tensor;
+  });
 
 DGL_REGISTER_GLOBAL("distributed.rpc._CAPI_DGLRPCFastPull")
-.set_body([] (DGLArgs args, DGLRetValue* rv) {
-  // Input
-  std::string name = args[0];
-  int local_machine_id = args[1];
-  int machine_count = args[2];
-  int group_count = args[3];
-  int client_id = args[4];
-  int service_id = args[5];
-  int64_t msg_seq = args[6];
-  std::string pickle_data = args[7];
-  NDArray ID = args[8];
-  NDArray part_id = args[9];
-  NDArray local_id = args[10];
-  NDArray local_data = args[11];
-  // Data
-  dgl_id_t ID_size = ID.GetSize() / sizeof(dgl_id_t);
-  dgl_id_t* ID_data = static_cast<dgl_id_t*>(ID->data);
-  dgl_id_t* part_id_data = static_cast<dgl_id_t*>(part_id->data);
-  dgl_id_t* local_id_data = static_cast<dgl_id_t*>(local_id->data);
-  char* local_data_char = static_cast<char*>(local_data->data);
-  std::vector<dgl_id_t> local_ids;
-  std::vector<dgl_id_t> local_ids_orginal;
-  std::vector<int64_t> local_data_shape;
-  std::vector<std::vector<dgl_id_t> > remote_ids(machine_count);
-  std::vector<std::vector<dgl_id_t> > remote_ids_original(machine_count);
-  // Get row size (in bytes)
-  int row_size = 1;
-  for (int i = 0; i < local_data->ndim; ++i) {
-    local_data_shape.push_back(local_data->shape[i]);
-    if (i != 0) {
-      row_size *= local_data->shape[i];
+  .set_body([](DGLArgs args, DGLRetValue* rv) {
```

Comment:
could you please not change the indent? it's very difficult for me to see what you changed.

Reply:
fixed. It's a mistake